app/proto/events.proto

```diff
@@ -294,6 +294,8 @@ message ListAllEventsReq {
 
   uint32 page_size = 2;
   string page_token = 3;
+  
+  bool include_cancelled = 4;
```

Comment:
Please move this above the `page_{size,token}`

Reply:
Done! Just out of curiosity, what does moving the location up do?

Reply:
Nothing, but it's just clearer since the pagination stuff should come after all the content :)